examples/skr/README.md

```diff
@@ -31,11 +31,15 @@ Once the key vault resource is ready, the user can import `RSA-HSM` or `oct-HSM`
 
 `go run <parent_dir>/tools/securitypolicydigest/main.go -p <base64-std-encoded-string-of-security-policy>`
 
+And the AAD token with permission to AKV/mHSM can be obtained with the following command: 
+
+`az account get-access-token --resource https://managedhsm.azure.net` 
+
 Once the `importkeyconfig.json` is updated, execute the following command:
 
 `cd <parent_dir>/tools/importkey`
 
-`go run main.go -c <parent_dir>/examples/skr/importkeyconfig.json`
+`go run main.go -c <parent_dir>/examples/skr/importkeyconfig.json -kh <RSA-HSM/oct-HSM key>`

```

Comment:
The kh is only for a hex key ... please replace with -kh <hexstring encoding of oct-HSM key>